server_src/runners/state.js

```diff
@@ -3228,14 +3233,28 @@ class Runner {
       profile.status = 900;
       sett.status = 900;
       if (
-        settings.printer.printerURL !== "" &&
-        settings.printer.printerURL !== farmPrinters[index].printerURL
+        settings.printer.printerURL.toString() !== "" &&
+        settings.printer.printerURL.toString() !==
+          farmPrinters[index].printerURL.toString()
```

Comment:
This hurts my eyes

Reply:
Poor planning on sending shit to the backend xD

Reply:
I dont think you're correct here as well. 

Please choose what applies:
1) `settings.printer.printerURL` is not a string
2) `farmPrinters[index].printerURL` is not a string

If either is the case, then
- what are they?
- why are they that value?
- how does converting a string to string help?